MoltenVK/MoltenVK/GPUObjects/MVKPixelFormats.mm

```diff
@@ -1281,8 +1298,53 @@
 	addMTLVertexFormatCapabilities( tvOS_GPUFamily1_v3, Short, Vertex );
 	addMTLVertexFormatCapabilities( tvOS_GPUFamily1_v3, Half, Vertex );
 	addMTLVertexFormatCapabilities( tvOS_GPUFamily1_v3, UChar4Normalized_BGRA, Vertex );
+
+	// Disable for simulator last.
+#if MVK_TVOS_SIMULATOR
+	disableAllMTLPixelFormatCapabilities(R8Unorm_sRGB);
+	disableAllMTLPixelFormatCapabilities(RG8Unorm_sRGB);
+	disableAllMTLPixelFormatCapabilities(B5G6R5Unorm);
+	disableAllMTLPixelFormatCapabilities(A1BGR5Unorm);
+	disableAllMTLPixelFormatCapabilities(ABGR4Unorm);
+	disableAllMTLPixelFormatCapabilities(BGR5A1Unorm);
+
+	disableAllMTLPixelFormatCapabilities(BGRA10_XR);
+	disableAllMTLPixelFormatCapabilities(BGRA10_XR_sRGB);
+	disableAllMTLPixelFormatCapabilities(BGR10_XR);
+	disableAllMTLPixelFormatCapabilities(BGR10_XR_sRGB);
+
+	disableAllMTLPixelFormatCapabilities(GBGR422);
+	disableAllMTLPixelFormatCapabilities(BGRG422);
+
+	disableMTLPixelFormatCapabilities(RGB9E5Float, ColorAtt);
+
+	disableMTLPixelFormatCapabilities(R8Unorm_sRGB, Write);
+	disableMTLPixelFormatCapabilities(RG8Unorm_sRGB, Write);
```

Comment:
Good catch. 

But I think I'll leave them in anyway because the docs are worded as two separate rules:

- Don't use the following pixel formats: `MTLPixelFormatR8Unorm_sRGB`...
- You can't write to sRGB textures in Simulator.

So if they relax one but not the other in the future, we'll still be covered correctly.